bot/controllers/userAdd.js

```diff
@@ -0,0 +1,28 @@
+const BaseController = require('../baseController.js');
+const Command = require('../baseCommand.js');
+
+class UserAdd extends BaseController {
```

Comment:
UserAdd is slightly ambiguous. I understand your intent (based upon the context of the assignment) but consider another dev looking at this code without that context, "UserAdd" seems more closely aligned to adding users to a database or our system.